solutions/64.js

```diff
@@ -0,0 +1,40 @@
+// Manik Sachdeva: msach22
+// reverse word order in a sentence
+// Input: "I am the king"
+// Output: "king the am I"
+/*
```

Comment:
also, this comment should be put right above solution function, because it describes the solution function